### PR TITLE
requirements: Use compatible versions

### PR DESCRIPTION
**Pull request by sils**

### CODE REVIEW

File: pyprint/VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.2.2
```
Comment (AbdealiLoKo) [inline pyprint/VERSION]: shouldnt we use dev999... ?

**Comment by AbdealiLoKo** [comment]
ack 54a5ddb 0d82775 ce91090 0fb5d34

**Comment by AbdealiLoKo** [comment]
I think version for dev should be 9999... though. But as we havent discussed this, acking it for now

**Comment by AbdealiLoKo** [comment]
If you do change it to dev999 you can reack it yourself though ^_^

**Comment by sils** [comment]
I'm not going to do a rebuild and wait another half hour

**Comment by sils** [comment]
@rultor merge

**Pull request merged**